.github/workflows/integrations.yml

```diff
@@ -70,10 +70,6 @@ jobs:
       - name: Test BAL
         run: |
           python -m pip install external/BAL
```

Comment:
[consider] Combine `pip install`s into a single line (pretty sure that'll work) and do a `grep` test to ensure `openassetio`/`openassetio-mediacreation` is definitely not overwritten?

Actually, that might be complicated by Windows unless/until we use `bash` throughout, i.e. #1114

Reply:
Do we care if `media-creation` is?